packages/lesswrong/lib/components.ts

```diff
@@ -192,6 +192,7 @@ importComponent("PostsItem2MetaInfo", () => require('../components/posts/PostsIt
 importComponent("PostsTitle", () => require('../components/posts/PostsTitle'));
 importComponent("PostsPreviewTooltip", () => require('../components/posts/PostsPreviewTooltip'));
 importComponent("PostsPreviewTooltipSingle", () => require('../components/posts/PostsPreviewTooltipSingle'));
+importComponent("PostsPreviewTooltipSingleWithComment", () => require('../components/posts/PostsPreviewTooltipSingle'));
```

Comment:
The previous line already imports the file. When the previous component gets used, the file gets executed and this component is also added to the components object. It's nevertheless good to add this line. Without it, if this component was used before the previous component, it would fail. (That's my 85% confident understanding.)

Reply:
It definitely wasn't working in this case until I added the second line.

Reply:
You were probably using before the previous component! I assumed it was working before.